### PR TITLE
Regenerated golang SDK via local full build

### PR DESCRIPTION
**Pull request by EricWittmann**

### CODE REVIEW

File: go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go [hunk 1]
```diff
@@ -10,13 +10,11 @@ type AdminRequestBuilder struct {
 }
 
 // ArtifactTypes the list of artifact types supported by this instance of Registry.
-// returns a *ArtifactTypesRequestBuilder when successful
```
Comment (andreaTP) [inline go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go:13]: this diff is really strange, is something going wrong with Kiota versions?
I just regenerated Go a couple of days ago.

Comment (EricWittmann) [reply go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go:13]: I dunno - I opened the PR to see what the diff was and didn't have a chance to look right away.  But now looking at the diff, it's all just comments!  wtf?  Environmental differences?

Comment (andreaTP) [reply go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go:13]: > Environmental differences?

Shouldn't ...

Comment (EricWittmann) [reply go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go:13]: :shrug:

**Pull request closed without merging**